### PR TITLE
bro

### PR DESCRIPTION
**Pull request by OttoDyva**

### CODE REVIEW

**Review by BekhanM — APPROVED** [review]
wrong one dont click

File: src/main/java/app/Main.java [hunk 1]
```diff
@@ -9,6 +10,8 @@
 
 public class Main
 {
+
```
Comment (tjorne) [inline src/main/java/app/Main.java:13]: why the newlines?

File: src/main/java/app/Main.java [hunk 2]
```diff
@@ -23,12 +26,19 @@ public static void main(String[] args)
         Javalin app = Javalin.create(config -> {
             config.staticFiles.add("/public");
             config.fileRenderer(new JavalinThymeleaf(ThymeleafConfig.templateEngine()));
+            config.staticFiles.add("/templates");
         }).start(7070);
 
         // Routing
-
         app.get("/", ctx -> ctx.render("index.html"));
+        app.get("/join", ctx -> ctx.render("jeopardy/join.html"));
```
Comment (tjorne) [inline src/main/java/app/Main.java:34]: why not add this to the JeopardyController.addRoutes()?

File: src/main/java/app/Main.java [hunk 3]
```diff
@@ -23,12 +26,19 @@ public static void main(String[] args)
...
         UserController.addRoutes(app, connectionPool);
         TimeZonesController.addRoutes(app, connectionPool);
+        JeopardyController.addRoutes(app, connectionPool);
+
+
```
Comment (tjorne) [inline src/main/java/app/Main.java:39]: why the newlines?

File: src/main/resources/public/css/jeopardy.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+.task-table td, .task-table th {
+    border: 3px solid #000000;
+    padding: 40px;
+    text-align: center;
+}
+.task-table th{
+    background-color: #efd69d;
+}
+
+/*.task-table tr:nth-child(even) {
```
Comment (tjorne) [inline src/main/resources/public/css/jeopardy.css:31]: unused code - should this be added to production?

File: src/main/resources/public/css/styles.css
Comment (tjorne) [inline src/main/resources/public/css/styles.css:1]: seems like it is only whitespace changes, but why are they here? 
I mean, it does look nicer than it did before, so perhaps one (or more) of you just had a case of OCD viewing this file? 😃

File: src/main/resources/templates/jeopardy/create.html
Comment (tjorne) [inline src/main/resources/templates/jeopardy/create.html:1]: is this file ready to be added to production?

**Review by tjorne — CHANGES_REQUESTED** [review]
Awesome project. 

There are some minor comments that should be addressed prior to this PR being approved and merged into production.

**Review by tjorne — APPROVED** [review]

**Pull request merged**